frontend/src/services/eam-api/eam-api.service.ts

```diff
@@ -62,6 +62,20 @@ class EAMApi {
     );
   }
 
+  public updateGroup(
+    id: string,
+    payload: EAMGroupCreateRequestDto,
+  ): Promise<EAMGroupCreateResponseDto> {
+    return this.#http.load(
+      joinItems(this.#apiPrefix, ApiPath.EAM, EAMApiPath.GROUPS, '/', id),
```

Comment:
use GroupsApiPath enum
```suggestion
      joinItems(`${this.#apiPrefix}${ApiPath.EAM}${EAMApiPath.GROUPS}${GroupsApiPath.ROOT}${id}`),
```